doc/user/structure.rst

```diff
@@ -31,15 +32,15 @@ range of targets, so are not expected to use the same naming conventions.
 RFI mask
 --------
 target
-    None
+    Empty string (i.e. a telescope-level model)
```

Comment:
I think I preferred this as it was. It's a zero-length list of components, rather than a single component which is an empty string.

Reply:
I also view it as a directory or hierarchical path, with an empty target being the root directory. The docs do say that it "may contain multiple components separated by slashes," which suggests to me that it is ultimately a path in string form and not a list. I'm happy to revert back to None.

I tried to convince Sphinx to put an empty string literal here in the same style as e.g. `meerkat`, but that is quite hard.